webui/react/src/shared/contexts/UIStore.ts

```diff
@@ -38,6 +42,42 @@ export type ActionUI =
   | { type: StoreActionUI.SetTheme; value: { darkLight: DarkLight; theme: Theme } }
   | { type: StoreActionUI.ShowUIChrome }
   | { type: StoreActionUI.ShowUISpinner };
+
+// CHECK: we don't really need this as a class but this
+// looked like the cleanest way to advertise the type in one package.
+export class UIActions {
```

Comment:
Just think of the `class` keyword in JS as light codegen. It's not fundamentally different than a regular object, it just adds some nice closures for you.